### ISSUE TITLE
combining with react-rnd

### ISSUE THREAD

**Issue by vbud** [aarondail/react-zoomable-ui#53]
This isn't an issue, more of a question. I'm using react-zoomable-ui to power a figma-like canvas with draggable, resizable frames.

Rather than using `<Pressable>` as shown on the example site, I was actually able to combine react-zoomable-ui with [react-rnd](https://github.com/bokuweb/react-rnd) by rendering the `<Rnd>` component inside a `<NoPanArea>`. The approach works really well and doesn't require anything hacky.

Not sure what kinds of use cases users of this library have, but if you think others would benefit, I'd be happy to put together a codesandbox link showing the approach. Let me know!

**Comment by aarondail** [comment]
Cool! Yeah I would definitely be interested in seeing a codesandbox if its not too much trouble? At the very least I could  adapt it to the examples to show that the two libraries can be used together cleanly.

Btw is the project you are using it for something you can share a link to? I'm just curious to see how this library is being used.

**Comment by vbud** [comment]
Here's a codesandbox: https://codesandbox.io/s/react-zoomable-ui-react-rnd-302vnz

I don't have a link to my project yet, but I will in the next month or two :).

**Comment by vbud** [comment]
Here's a link to an example file in the project I'm working on: https://composer.design/example_tables.

Think of it as a mash-up of a code editor and a design tool like figma, hence the need for a zoomable, pan-able canvas.

I'm seeing lots of canvas-y stuff these days... tldraw, natto, etc.

**Comment by aarondail** [comment]
Nice, and thats an interesting idea. Good luck with your project.

natto and tldraw are interesting too, especially natto. Ill have to take a closer look at that (seems vaguely similar to some ideas I have had too). Thanks for sharing.

**Comment by vbud** [comment]
Will close this for now, thanks for the conversation and for helping out with the bug earlier this year!

**Comment by aarondail** [comment]
Also fyi I checked out your react-rnd integration example, and thanks for posting that. Overall works surprisingly well, but there are a few minor problems I ran into where its not behaving exactly as I'd like after you zoom in/out. Sometimes the box jerks a bit after a drag, and sometimes drags/resizes don't quite track the cursor movement. I think those things are probably not too hard to fix but I'd like to do that first. I uh will try get to that at some point. :)
